### PR TITLE
Rocket Pods now require manual disassembly

### PR DESCRIPTION
**Pull request by Iamgoofball**
Bluespacing out simply wasn't metal enough tbh

### CODE REVIEW

**Comment by KorPhaeron** [comment]
@MrDoomBringer up to you

File: code/game/objects/structures/crates_lockers/closets/secure/bsdroppod.dm [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 //------------------------------------BLUESPACE DROP POD-------------------------------------//
 /obj/structure/closet/bsdroppod
-	name = "Bluespace Drop Pod"
```
Comment (XDTM) [inline code/game/objects/structures/crates_lockers/closets/secure/bsdroppod.dm]: you mean /obj/structure/closet/sdroppod surely?

**Comment by MrDoomBringer** [comment]
Nope from me. The sprites are slightly larger than 32x32 (to accommodate the crates inside), and all this will do is clutter cargo with a mess of ugly, overlapping sprites. Repairing damaged floor tiles is plenty work for the drop pods.

**Comment by Iamgoofball** [comment]
@MrDoomBringer negative, it looks fine visually. It shouldn't be cluttering cargo, they should be welding the pods apart right after receiving them. This also stacks well with the other change I made for the emags.

File: code/game/objects/structures/crates_lockers/closets/secure/bsdroppod.dm (hunk 1 above)
Comment (Iamgoofball) [reply code/game/objects/structures/crates_lockers/closets/secure/bsdroppod.dm]: no point in a repath

**Comment by Iamgoofball** [comment]
ps. idle in coderbus

**Comment by MrDoomBringer** [comment]
@Iamgoofball, positive, spamming droppods and having to deal with the damaged floortiles/hull breeches, then on top of that having to deal with these fucking enormous sprites sliding around everywhere blocking your path? This PR seems less interesting gameplay element and more frustrating clutter.

**also: right click menu picks up on the droppods even if you're not moused over them, also people are above droppod layer which looks gross, imo.**
image

**Comment by Iamgoofball** [comment]
Damaged floor tiles are piss easy to deal with, same with breaches. Welders are also super-available.

If you're paying for instant delivery, you shouldn't get to clean up after yourself instantly.

**Comment by Iamgoofball** [comment]
Anchor them if there's an issue with movement.

In fact, I'll do that now.

**Comment by Iamgoofball** [comment]
wait m8 they're already anchored, what are you talking about with them "sliding around"

**Comment by MrDoomBringer** [comment]
already anchored sil-

**Comment by MrDoomBringer** [comment]
anyways, another issue I have with this is that at some point, I'd love to add a laser designator type thing for late-game cargo memes. The burden for dealing with the drop pod should be on cargo, not on the department recieving them (as it currently is, with cargo having to move crates, stamp manifests, keep track of shit, etc)

**Comment by MrDoomBringer** [comment]
also >says to idle in coderbus
>doesnt reply to me in coderbus
:<

**Comment by Iamgoofball** [comment]
fugg sorry was busy

Re: laser designator, i'd say make that one bluespace it out I guess

**Comment by Iamgoofball** [comment]
oh

you pinged oranges, not me

**Comment by MrDoomBringer** [comment]
shit

fucking nicknames

**Comment by MrDoomBringer** [comment]
Anyways, I kinda like the idea of having more advanced droppods that bluespace out, so i guess im onboard with this then. !!progressiooon!!

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**